website/docs/components/app-footer/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,88 @@
+## Usage
+
+### When to use
+
+- On every screen of an application where universal links or content are required.
+
+## Order of content
+
+While the code allows flexibility in the ordering of links and content, we recommend the following structure for a consistent visual appearance across applications: 
+
+- Status link
```

Comment:
@andgen404 ?

Reply:
Hm, but for the (extra) links, items, and Copyright, they would be written like:
- StatusLink
- LegalLink
- Links
- Items
- Copyright

Reply:
I think you can probably leave the other items as they are (since they're a bit more generic). The StatusLink, and LegalLinks are essentially named items that we're introducing so using consistent language for them might be beneficial.

Reply:
Okay, I see. I'll make those changes 👍